examples/general/ami/post_installation.md

```diff
@@ -0,0 +1,126 @@
+# AMI 
+
+You need to configure head node and compute nodes via post-installation actions. The following are the actions required:
+
+1. Create a `install_neuron.sh` script that has the following content:
+
+```
```

Comment:
Need a `#!/bin/bash` in the shell scripts as well. Both for install_neuron.sh and install_python_venv.sh

Reply:
Thanks will update in new commit

Reply:
did you fix?

Reply:
yes fixed